src/lib.rs

```diff
@@ -409,9 +427,21 @@ impl<'a, Key: Ord + Clone, Value> Entry<'a, Key, Value> {
 #[cfg(test)]
 mod test {
     use rand;
-    use std::thread;
     use std::time::Duration;
 
+    #[cfg(feature = "fake_clock")]
+    fn sleep(time: Duration) {
+        use fake_clock::FakeClock;
+        // adding one millisecond to simulate waiting a tiny bit longer than requested
```

Comment:
Yes, if that's necessary for the test to pass, i.e. if the semantics are strict inequality rather than inequality, I think that should be reflected in the tests.